### ISSUE TITLE
Requesting a sync for the FineTuning API

### ISSUE THREAD

**Issue by sashirestela** [openai/openai-openapi#217]
Please, could you sync the last Fine Tuning API changes?

I cannot find the specifications for:
- `integrations` field in the request for creating a fine-tuning job.
- `seed` field in the request for creating a fine-tuning job.
- endpoint `list checkpoints` for a fine-tuning job.
